worlds/sm/__init__.py

```diff
@@ -279,14 +280,14 @@ def APPrePatchRom(self, romPatcher):
         # first apply the sm multiworld code patch named 'basepatch' (also has empty tables that we'll overwrite),
         #  + apply some patches from varia that we want to be always-on.
         # basepatch and variapatches are both generated from https://github.com/lordlou/SMBasepatch
-        romPatcher.applyIPSPatch(os.path.join(os.path.dirname(__file__),
-                                              "data", "SMBasepatch_prebuilt", "multiworld-basepatch.ips"))
-        romPatcher.applyIPSPatch(os.path.join(os.path.dirname(__file__),
-                                              "data", "SMBasepatch_prebuilt", "variapatches.ips"))
+        romPatcher.applyIPSPatch("/".join((str(appDir),
```

Comment:
AP injects a `__file__` attribute on your world, which might be what you want? pkgutil would be better

Reply:
I aim to minimize changes done to the base code to ease future update of VARIA. Using pkgutil.get_data would require more changes. As for World.__file__, I'll use it here but trying to import from __init__ in deeper packages results in circular dependencies. I would need to pass down the value, which I'm not fond of.

Maybe I'm missing something else?